src/crawlee/browsers/_base_browser_controller.py

```diff
@@ -59,13 +59,13 @@ def browser_type(self) -> BrowserType:
     @abstractmethod
     async def new_page(
         self,
-        page_options: Mapping[str, Any] | None = None,
+        browser_new_context_options: Mapping[str, Any] | None = None,
         proxy_info: ProxyInfo | None = None,
     ) -> Page:
         """Create a new page with the given context options.
 
         Args:
-            page_options: Options to configure the new page.
+            browser_new_context_options: Options to configure the new context used for creating new page.
```

Comment:
Could we use the same docstring as in the PW crawler (except for the "This option should not be used if `browser_pool` is provided.")? This is also end-user interface.